src/App.tsx

```diff
@@ -1,24 +1,193 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { TodoStatus } from './types/TodoStatus';
+import { ErrorMessage } from './types/ErrorMessage';
+import {
+  getTodos, USER_ID, createTodo, removeTodo,
+} from './api/todos';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
 
-const USER_ID = 0;
+const statusTodos = (todos: Todo[], filterBy: TodoStatus) => {
```

Comment:
My assumption better name will be `getVisibleTodos`
```suggestion
const getVisibleTodos = (todos: Todo[], filterBy: TodoStatus) => {
```